app/src/main/kotlin/com/brianco/simpledaycounter/DayCounterWidgetConfigureActivity.kt

```diff
@@ -58,6 +60,9 @@ class DayCounterWidgetConfigureActivity : Activity() {
     val backgroundColorContainer = findViewById<ViewGroup>(R.id.background_color_circle_container)
     val addWidgetButton = findViewById<Button>(R.id.configure_add)
 
+    previewCounterLoading.visibility = View.GONE
+    previewCounter.visibility = View.VISIBLE
```

Comment:
The DayCounterWidgetConfigureActivity could use a different layout resource than the widget, but this is good enough.